tests/allow-privileged-import.earth

```diff
@@ -1,8 +1,8 @@
-VERSION 0.6
+VERSION 0.7
 FROM alpine:3.15
 RUN apk add libcap # for capsh
 
-IMPORT --allow-privileged github.com/earthly/test-remote/privileged:main
+IMPORT --allow-privileged github.com/earthly/test-remote/privileged:45359a92acdd7ef27d561d0836099cdacc508e40
```

Comment:
Will we need to update these tags after merge? (this is more of a process question than a review comment)